listenbrainz_spark/path.py

```diff
@@ -1,12 +1,34 @@
 import os
+import config
+
+# Parent directory containing listens and data driven from ListenBrainz.
+LISTENBRAINZ_DATA_DIRECTORY = os.path.join('/', 'data', 'listenbrainz')
```

Comment:
Are there places in the codebase we should change to use this? I think we should do that in this PR as well.